CHANGELOG.rst

```diff
@@ -6,6 +6,135 @@
 Synapse Changelog
 *****************
 
+v2.174.0 - 2024-07-06
+=====================
+
+Automatic Migrations
+--------------------
+- Renormalize ``ou:position:title``, ``ou:conference:name``, and
+  ``ou:conference:names`` secondary properites.
+  (`#3701 <https://github.com/vertexproject/synapse/pull/3701>`_)
+- Migrate instances of ``econ:currency`` properties into their own nodes.
+  (`#3790 <https://github.com/vertexproject/synapse/pull/3790>`_)
+- Add a Cortex storage migration to set the correct View iden value on all
+  Trigger definitions.
+  (`#3760 <https://github.com/vertexproject/synapse/pull/3760>`_)
+- See :ref:`datamigration` for more information about automatic migrations.
+
+Model Changes
+-------------
+- Add a new model, ``entity``, for modeling elements related to entity
+  resolution.
+  (`#3781 <https://github.com/vertexproject/synapse/pull/3781>`_)
+- Updates to the ``crypto``, ``econ``, ``files``, ``ou``, and ``pol`` models.
+  (`#3790 <https://github.com/vertexproject/synapse/pull/3790>`_)
+  (`#3781 <https://github.com/vertexproject/synapse/pull/3781>`_)
+
+  **New Forms**
+
+  ``econ:currency``
+    The name of a system of money in general use.
+
+  ``entity:name``
+    A name used to refer to an entity.
+
+  **New Properties**
+
+  ``crypto:key``
+    The form had the following properties added to it:
+
+    ``private:text``
+      Set only if the ``:private`` property decodes to ASCII.
+
+    ``public:text``
+      Set only if the ``:public`` property decodes to ASCII.
+
+  ``econ:acct:payment``
+    The form had the following properties added to it:
+
+    ``from:cash``
+      Set to true if the payment input was in cash.
+
+    ``to:cash``
+      Set to true if the payment output was in cash.
+
+    ``place``
+      The place where the payment occurred.
+
+    ``place:address``
+      The address of the place where the payment occurred.
+
+    ``place:latlong``
+      The latlong where the payment occurred.
+
+    ``place:loc``
+      The loc of the place where the payment occurred.
+
+    ``place:name``
+      The name of the place where the payment occurred.
+
+  ``pol:country``
+    The form had the following property added to it:
+
+    ``currencies``
+      The name of the place where the payment occurred.
```

Comment:
If this was auto-populated there may be a copy-pasta error. This should be more like "The currencies officially used by the country."

Reply:
copy-pasta error :spaghetti: